### DISCUSSION TITLE
Cant move files with long names

### DISCUSSION THREAD

**Discussion by Amrit-Padda** [Serene-Arc/bulk-downloader-for-reddit#445] [Ideas]
After downloading, if a filename is too long, windows doesnt let you copy the file. 

Maybe concat names above a certain length?

**Comment by aliparlakci** [comment]
Yes, in Windows, a full path of a directory can be 260 characters long at max. You can easily disable it. It will fix the problem.

If you don't want to disable it, renaming the long files can be done with a super simple script.

Another option would be specifying a [`--file-scheme`](https://github.com/aliparlakci/bulk-downloader-for-reddit#folder-and-file-name-schemes) to exclude the post title from file names.

However, implementing it inside bdfr would only bring unnecessary complexity and bloat the codebase. I suggest you to consider the options above.

**Reply by Serene-Arc** [reply]
Adding onto this, we do limit the filenames so that they can be downloaded. There is no conceivable way for us to limit the filename so that you can copy them wherever you want, it just doesn't work that way. Where you can copy it is dependent on the location; the more nested the folder structure, the smaller the filename can be. Since we already limit them to the maximum allowed, you'll have to do anything more.
